front/src/modules/simulationResult/components/SimulationResultsMap/RenderItinerary.tsx

```diff
@@ -23,15 +28,31 @@ export default function RenderItinerary(props: RenderItineraryProps) {
     'line-color': '#158DCF',
   };
 
-  // We know we have a geometry so we have at least 2 coordinates
-  const [startLongitude, startLatitude] = geojsonPath.geometry.coordinates.at(0)!;
-  const [endLongitude, endLatitude] = geojsonPath.geometry.coordinates.at(-1)!;
+  if (!pathItemsCoordinates || pathItemsCoordinates.length < 2) {
+    return null;
+  }
+
+  const [originLongitude, originLatitude] = pathItemsCoordinates[0];
+  const [destinationLongitude, destinationLatitude] =
+    pathItemsCoordinates[pathItemsCoordinates.length - 1];
```

Comment:
We try to always use this syntax now. You can add the `!` because at this point we know there is at least 2 path items.

```suggestion
  const [originLongitude, originLatitude] = pathItemsCoordinates.at(0)!;
  const [destinationLongitude, destinationLatitude] =
    pathItemsCoordinates.at(-1)!;
```

Reply:
done